src/shared/extensionUtilities.ts

```diff
@@ -148,7 +147,7 @@ function convertExtensionRootTokensToPath(
  */
 export function isDifferentVersion(context: vscode.ExtensionContext, currVersion: string = pluginVersion): boolean {
     const mostRecentVersion = context.globalState.get<string>(mostRecentVersionKey)
-    if (mostRecentVersion && semver.valid(mostRecentVersion) && mostRecentVersion === currVersion) {
+    if (mostRecentVersion && mostRecentVersion === currVersion) {
```

Comment:
Update the function heading to remove semver comments

Reply:
✔️